File: ReSharper.FSharp/test/data/cache/csharpResolve/Methods 02, compiled name.cs
using System.Collections.Generic;
using static Module;

public class Class1
{
  public Class1()
  {
    var t = new T();
    int m = t.InstanceMethod();
    int sm = T.StaticMethodCompiled();

    int oInt = t.OverloadInt(123, 123);
    string oString = t.OverloadString("123", "123");

    int oStaticInt = T.StaticOverloadInt(123, 123);
    string oStaticString = T.StaticOverloadString("123", "123");
  }
}

File: ReSharper.FSharp/test/data/cache/csharpResolve/Methods 02, compiled name.fs
module Module

type T() =
    [<CompiledName("InstanceMethod")>]
    member x.Method() = 123

    [<CompiledName("StaticMethodCompiled")>]
    static member StaticMethod() = 123

    [<CompiledName("OverloadInt")>]
    member x.Overloads(a, b) = a + b

    [<CompiledName("OverloadString")>]
    member x.Overloads(a: string, b: string) = a + b

    [<CompiledName("StaticOverloadInt")>]
    static member StaticOverloads(a, b) = a + b

    [<CompiledName("StaticOverloadString")>]
    static member StaticOverloads(a: string, b: string) = a + b